en/messages.json

```
@@ -352,5 +352,14 @@
   "telemetrySettingsCheckboxLabel": {
     "message": "This data helps us improve features and performance. It includes data about your device, hardware configuration, and how you use the Mozilla VPN extension.",
     "description": "Checkbox label on the settings page"
+  },
+  "messageSplitTunnelHeader": {
+    "message": "Re-enable Firefox protection",
```

```suggestion
    "message": "Enable protection for Firefox",
```

Based on Slack conversation